region/crateria/east/East Ocean.json

```diff
@@ -159,25 +159,63 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Mockball (Space Jump)",
+      "requires": [
+        "canPreciseSpaceJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 18,
```

Comment:
This looks like missing Gravity?

Reply:
I can't figure out how to reach the door platform.
Landing on the first platform and springballing to the door is much easier.

Reply:
Good catch, added Gravity requirement.

Landing on the first platform is much easier, yeah. I split off a separate strat for that now, since it requires Spring Ball to make it to the door from there. After some more practice, found how to make it all the way to the door while still spinning, which seems very precise. It will take some more work to figure out momentum constraints for all these strats